src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -4497,6 +4498,7 @@ void CalcHeatEmissionReport(EnergyPlusData &state)
         if (state.dataHeatingCoils->HeatingCoil(iCoil).HCoilType_Num == DataHVACGlobals::Coil_HeatingGas_MultiStage ||
             state.dataHeatingCoils->HeatingCoil(iCoil).HCoilType_Num == DataHVACGlobals::Coil_HeatingGasOrOtherFuel) {
             state.dataHeatBal->SysTotalHVACRejectHeatLoss += state.dataHeatingCoils->HeatingCoil(iCoil).FuelUseLoad +
+                                                             state.dataHeatingCoils->HeatingCoil(iCoil).ElecUseLoad +
```

Comment:
This one is hard to answer not knowing the intent, as is the change at line 4449. As I look at each of these summations in CalcHeatEmissionReport (and read the Purpose comment at the top, which didn't help a bit), I see things like condenser energy, the energy to reject heat from the building, or exhaust air systems energy, or refrigeration system energy from DX coil heat recovery, things that are obviously energy use to reject heat from the building. Then I look at this item. The amount of energy the heating coil uses to heat the building. How is that heat rejection? Maybe the fuel energy leaving the building that was not used to heat the building, as hot exhaust gases, but all of the fuel energy? Some of that energy entered the building. Seems to me there should be a (1 - Effic) here on FuelUseLoad. HeatingCoilLoad went into the building, the rest was exhausted.

            heatingCoil.FuelUseLoad = HeatingCoilLoad / Effic;

You have to wonder if when each item was added here if it was added correctly in the spirit of this report. To your question, is gas heating coil parasitic a `SysTotalHVACRejectHeatLoss`, I would have to agree, that energy was not added to the building, but consumed because of heating the building. Inclusion of each of these items is a report definition issue.

Reply:
I decided to revert this change to avoid CI simulation diffs in this PR. I opened a [GitHub issue](https://github.com/NREL/EnergyPlus/issues/10123) if we think this should be addressed.